indico/modules/events/abstracts/settings.py

```diff
@@ -53,6 +68,14 @@ class BOACorrespondingAuthorType(RichEnum):
 }
 
 
+BOALinkFormatting.__titles__ = {
+    BOALinkFormatting.frame: _('Frame'),
+    BOALinkFormatting.colorlinks: _('Color Links'),
+    BOALinkFormatting.ocgcolorlinks: _('Color Links for screen only'),
+    BOALinkFormatting.ocgx2colorlinks: _('Color Links for screen only (alt., use ocgx2)'),
```

Comment:
I'm still unsure about this...

- if this is better than the option above and is supported with texlive 2017 and newer, let's just keep this one, without the `(...)` part in the title
- it's probably still confusing users - i guess even most LaTeX users have never heard of "ocgx2"